api/CustomElementRegistry.json

```diff
@@ -149,23 +149,13 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/CustomElementRegistry/define",
           "support": {
-            "webview_android": [
-              {
-                "version_added": "66",
-                "notes": "Support for 'Customized built-in elements' as well."
-              },
-              {
-                "version_added": "54",
-                "notes": "Support for 'Autonomous custom elements' only."
-              }
-            ],
             "chrome": [
               {
                 "version_added": "66",
```

Comment:
Shouldn’t this be `67` [like above](https://github.com/mdn/browser-compat-data/pull/1708/files#diff-c0c39ea3ac0b61fea9f2765a6263d4e5R81)?